packages/react/src/api/ThreadListRuntime.ts

```
@@ -95,10 +134,26 @@ export class ThreadListRuntimeImpl implements ThreadListRuntime {
 
+  public readonly main: ThreadRuntime;
+
   public get mainItem() {
     return this._mainThreadListItemRuntime;
   }
 
+  public getById(threadId: string) {
+    return new this._runtimeFactory(
+      new NestedSubscriptionSubject({
+        path: {
+          ref: "threads[threadId=" + JSON.stringify(threadId) + "]",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)